### DISCUSSION TITLE
Unused dependency

### DISCUSSION THREAD

**Discussion by tabuna** [orchestral/testbench#301] [General]
Hi everyone. I noticed that recently in my update, there was a package for a commercial product Ray.
Is there a reason for this other than advertising?  I have not found its use in the organization's projects (Maybe I was [looking bad](https://github.com/search?q=org%3Aorchestral+LaravelRay)? ). What do you think about changing its status depending on[ the recommended ones](https://getcomposer.org/doc/04-schema.md#suggest)?

```json
{
    "suggest": {
        "spatie/laravel-ray": "Debug with Ray to fix problems faster in Laravel apps",
    }
}
```

I would really appreciate an answer. Best wishes

**Comment by crynobone** [comment]
> Is there a reason for this other than advertising? 

I added it because I use it every day, Did I get paid for it? nope.

**Reply by tabuna** [reply]
Oh, I didn't mean to say that you got paid for this ad. Advertising can also be on a voluntary basis.
 So what do you think about translating from dependencies to suggest?

**Reply by crynobone** [reply]
Testbench is setup ready to go with all the required and suggested feature. If we want to move to "suggested deps" then `symfony/yaml`, `fakerphp/faker` and `mockery/mockery` should be done the same way too. 

But then again you can opt to use `orchestra/testbench-core` with does exactly what you're looking for.

**Reply by tabuna** [reply]
Unfortunately, `orchestral/testbench-dusk` depends on `orchestral/testbench`

**Reply by Ilyes512** [reply]
I also have an issue with the ray dependency... I wanted to test `guzzlehttp/psr:2.0.0-beta1`, but because of ray's sub-dependency `zbateson/mail-mime-parser` it conflicts.

My opinion is that Ray is not the same as using `symfony/yaml`, `fakerphp/faker` and `mockery/mocker`.
